applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -81,6 +80,48 @@ static le_audio_receive_cb receive_cb;
 static bool init_routine_completed;
 static bool paused;
 
+static struct bt_csip_set_member_svc_inst *csip;
+/* Advertising data for peer connection */
+static uint8_t csip_rsi_adv_data[BT_CSIP_RSI_SIZE];
+
+#define CSIP_SET_SIZE 2
```

Comment:
Should this be a Kconfig?

Reply:
I would suggest we use MACRO in here to make it simple, 2 means there's only 2 devices (L and R) in the set.